quint/src/flattening.ts

```diff
@@ -459,6 +468,22 @@ class Flatenner {
   }
 }
 
+function inlineTypeAliases(modules: QuintModule[], table: LookupTable): { modules: QuintModule[]; table: LookupTable } {
+  const modulesWithInlinedAliases = modules.map(m => inlineAliases(table, m))
+  const tableWithInlinedAliases = new Map(
+    [...table.entries()].map(([id, def]) => {
+      if (!def.typeAnnotation) {
+        return [id, def]
+      }
+
+      const inlinedType = inlineAliasesInType(table, def.typeAnnotation)
+      return [id, { ...def, typeAnnotation: inlinedType }]
```

Comment:
This constructs a fresh `AliasInliner` on each table entry.

Would it make more sense to refactor this code into a function in `aliasInliner.ts`?
That would give better symmetry with the call for modules.